lib/src/doc/field.rs

```diff
@@ -35,7 +35,11 @@ impl<'a> Document<'a> {
 				let def = match &fd.default {
 					Some(v) => Some(v),
 					_ => match &fd.value {
-						Some(v) if v.is_static() => Some(v),
+						Some(v) => match &v {
+							Value::Function(f) if f.args().is_empty() => Some(v),
```

Comment:
```suggestion
```

It could be better to remove this and instead, alter the `v.is_static()` function:

`lib/src/sql/value.rs`

```rs
	/// Checks whether this value is a static value
	pub(crate) fn is_static(&self) -> bool {
		match self {
			Value::None => true,
			Value::Null => true,
			Value::Bool(_) => true,
			Value::Bytes(_) => true,
			Value::Uuid(_) => true,
			Value::Number(_) => true,
			Value::Strand(_) => true,
			Value::Duration(_) => true,
			Value::Datetime(_) => true,
			Value::Geometry(_) => true,
			Value::Array(v) => v.iter().all(Value::is_static),
			Value::Object(v) => v.values().all(Value::is_static),
			Value::Constant(_) => true,
			_ => false,
		}
	}
```

`lib/src/sql/function.rs`

```rs
	/// Check if this function is a custom function
	pub fn is_custom(&self) -> bool {
		matches!(self, Self::Custom(_, _))
	}

	/// Check if this function is a scripting function
	pub fn is_script(&self) -> bool {
		matches!(self, Self::Script(_, _))
	}
	
	/// Check if this function has static arguments
	pub fn is_static(&self) -> bool {
		match self {
			Self::Normal(_, a) => a.iter().all(Value::is_static),
			_ => false,
		}
	}
```